pages/index.tsx

```diff
@@ -99,6 +126,9 @@ export const getServerSideProps: GetServerSideProps = async ({ req, res, locale,
     urlPrefixBpo: process.env.URL_PREFIX_BPO ?? '',
   }
 
+  // Set to 'enabled' to include Google Analytics code
+  const ENABLE_GOOGLE_ANALYTICS = process.env.ENABLE_GOOGLE_ANALYTICS ?? ''
```

Comment:
Do we need to be alerted on whether this env var is missing? If so, there is code in `queryApiGateway.ts` that could be re-used or generalized.

Reply:
Oh interesting! I guess ideally we could be alerted if it's missing in production, and only in production. I don't think we currently have a way of detecting whether we're in production (although it's easy enough to add an env var for that), so I'll create a ticket to alert if it's missing in prod, how's that sound?

Reply:
We do! We're doing that for the missing unique number.

Reply:
(talked to Rocket, we don't have a way *in our codebase* of detecting whether we're in prod)

created: https://github.com/cagov/ui-claim-tracker/issues/460